components/mjs/a11y/util-lab.js

```diff
@@ -0,0 +1,23 @@
+import {Loader} from '#js/components/loader.js';
+import '../input/mml/init.js';
+import {Sre} from './sre/sre.js';
+import './semantic-enrich/semantic-enrich.js';
+import './explorer/explorer.js';
+//import MathMaps from '#js/a11y/mathmaps.js';
+//import base from 'speech-rule-engine/lib/mathmaps/base.json' assert { type: 'json' };
+//import en from 'speech-rule-engine/lib/mathmaps/en.json' assert {type: 'json' };
+//import nemeth from 'speech-rule-engine/lib/mathmaps/nemeth.json' assert {type: 'json' };
```

Comment:
So we no longer build with default locales?

Reply:
As you note below, this `util-lab.js` file is only used by the v4 lab.  The reason is that node, webpack, and the browser each need the json files to be handled differently.  Webpack requires the `assert { type: 'json' }` hint, but node won't use that without an experimental command line argument, while the browser won't import json at all.  So I needed to make a separate util file for each case.  This is the browser one, which doesn't pre-load the locales, but will use its own loading mechanism to do so when needed.  The `util-pack.js` file uses includes the the `assert` hints, while the `util.js` file used by node when running node-main with component source files uses `require` to avoid the `assert` issue with `import`.  This works in both mjs and cjs settings since our `node-main.mjs` sets up `require` so that SRE will be able to load files, so we take advantage of that here.